src/vic3save/src/savefile.rs

```diff
@@ -0,0 +1,268 @@
+use crate::stats::Vic3CountryStats;
+use crate::Vic3Date;
+use serde::de;
+use serde::de::Unexpected;
+use serde::{de::DeserializeOwned, Deserialize, Deserializer};
+use std::collections::HashMap;
+use std::fmt;
+use std::hash::Hash;
+use std::marker::PhantomData;
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct MetaData {
+    pub version: String,
+    pub game_date: Vic3Date,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct Counters {
+    week: i32,
```

Comment:
I just ran this on a save and `week` appears to occasionally be omitted, so you'll want to either use `#[serde(default)]` to assign a default value of 0 or to use `Option<i32>`